recipes/knotinframe/meta.yaml

```diff
@@ -1,37 +1,42 @@
 {% set version = "2.3.1" %}
-{% set sha256 = "d9ab68329e48709cffbf5ed775f905bdddebab470d70836bb522993b971d48af" %}
+{% set packagename = "knotinframe" %}
+{% set sha256 = "0c5b7d91893d0a23189f4af3602e18f110f030eac49a3658f7acede23596fde0" %}
 
 package:
-  name: knotinframe
-  version: {{ version }}
+  name: {{ packagename }}
+  version: 2.2.0
```

Comment:
Why are you removing the {{version}} template and downgrading from 2.3.1 to 2.2.0 here?

Reply:
Hi @epruesse thanks for your review!
The {{version}} variable shall address the release version of the upstream repository. This repository serves not only knotinframe, but also rnashapes, rnaalishapes, pkiss, palikiss, rapidshapes and acms. 

The version of the software KnotInFrame was previously confused with the upstream repo version and should have been "2.1.0" as defined in https://github.com/jlab/fold-grammars/blob/5b46de8bb107d9b5cf4955136e7ccffdaf0bca75/Misc/Applications/lib/foldGrammars/Settings.pm#L15 and was actually bumped to 2.2.0 by me recently.

My idea is that {{version}} of the upstream package will help recognizing novel releases of the upstream repo and thus help auto bump bioconda packages.